<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>C on The Journeyman Modeler</title><link>http://jmodeler.github.io/tags/c/</link><description>Recent content in C on The Journeyman Modeler</description><generator>Hugo</generator><language>en</language><lastBuildDate>Wed, 12 Apr 2023 13:43:59 -0400</lastBuildDate><atom:link href="http://jmodeler.github.io/tags/c/index.xml" rel="self" type="application/rss+xml"/><item><title>Resolving Errors When Loading the lldb Debugger</title><link>http://jmodeler.github.io/posts/resolving-errors-when-loading-the-lldb-debugger/</link><pubDate>Wed, 12 Apr 2023 00:00:00 +0000</pubDate><guid>http://jmodeler.github.io/posts/resolving-errors-when-loading-the-lldb-debugger/</guid><description>From time to time, I work with C code, and I use the lldb debugger to step through the code. Recently, after a Mac OS update, I started seeing this error when attempting to start the lldb debugger in terminal:
$ lldb Traceback (most recent call last): File &amp;#34;&amp;lt;string&amp;gt;&amp;#34;, line 1, in &amp;lt;module&amp;gt; File &amp;#34;/usr/local/Cellar/llvm/15.0.7_1/libexec/python3.11/site-packages/lldb/__init__.py&amp;#34;, line 96, in &amp;lt;module&amp;gt; import six ModuleNotFoundError: No module named &amp;#39;six&amp;#39; The error message says that a module (six) is missing.</description></item></channel></rss>